trento/migration/systemd-install.md

```diff
@@ -408,6 +404,12 @@ This is a basic guide for creating a self-signed certificate for use with Trento
    ```bash
    systemctl enable --now nginx
    ```
+   
+1.  Copy the generated trento.crt in a location accessible by nginx:
+   
+    ```bash
+    cp trento.crt /etc/nginx/ssl/certs
```

Comment:
We should be consistent, either we do the command like in line [355](https://github.com/SUSE/doc-unversioned/pull/185/files#diff-b35cbb9532a7b6cd6c3420c3f6c10f9423aab6baa541f33a5e544fa5fe40a709R355) or we change it there. 

The difference is that `cp trento.key /etc/ssl/private/trento.key` specifies a target file name, potentially renaming the file, while `cp trento.key /etc/ssl/private/` clearly copies the file into the directory without changing its name.

Reply:
```suggestion
    cp trento.crt /etc/nginx/ssl/certs/trento.crt
```